modules/apps/portal/portal-default-permissions-web/src/main/java/com/liferay/portal/defaultpermissions/web/internal/configuration/display/PortalDefaultPermissionsCompanyConfigurationScreenWrapper.java

```diff
@@ -63,7 +64,7 @@ public String getCategoryKey() {
 		@Override
 		public String getJspPath() {
 			return "/configuration" +
-				"/portal_default_permissions_company_configuration.jsp";
+				"/portal_default_permissions_configuration.jsp";
```

Comment:
why you did this change? I think the same should have been done for the label key

Reply:
Initially I thought I needed two different pages (one for company settings and one for group settings) but instead I managed to use only one for both (so the rename to a more "generic" page